src/hotspot/os_cpu/linux_ppc/javaThread_linux_ppc.cpp

```diff
@@ -24,6 +24,7 @@
  */
 
 #include "memory/metaspace.hpp"
+#include "os_linux.hpp"
```

Comment:
I think you should include `os_linux.inline.hpp` (according to the [style-guide](https://github.com/openjdk/jdk/blob/2e4d7d1846d846fd98201b9b3abeb7b91239a40d/doc/hotspot-style.md?plain=1#L163)).

Reply:
Not really? I think that rule only says "do not include `os_linux.hpp` if `os_linux.inline.hpp` is already included". We do not include `os_linux.inline.hpp`, so we should include just the `os_linux.hpp`.

Reply:
You're right. I misread this rule as a rule I expected to exist too (always prefer the *.inline.hpp over *.hpp).